src/cdk-experimental/popover-edit/edit-event-dispatcher.ts

```diff
@@ -62,6 +62,12 @@ export class EditEventDispatcher {
   /** A subject that emits mouse move events from the table indicating the targeted row. */
   readonly mouseMove = new Subject<Element|null>();
 
+  /**
+   * Tracks the currently disabled editable cells - edit calls will be ignored
+   * for these cells.
+   */
+  readonly disabledCells = new WeakSet<Element>();
```

Comment:
We have to use `WeakMap<Element, boolean>` until we drop IE11

Reply:
Done